admin/src/components/ContributionForm/index.tsx

```diff
@@ -1,171 +0,0 @@
-import React from 'react';
```

Comment:
This does not need to be brought in given that there is no reason to have admin created contributions going forward.